<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PageFrame</class>
 <widget class="QWidget" name="PageFrame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>492</width>
    <height>386</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>20</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>4</number>
   </property>
   <property name="margin">
    <number>4</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>4</number>
     </property>
     <item>
      <widget class="QCheckBox" name="useCheckBox">
       <property name="text">
        <string>Использовать раздел</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="nameEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QWidget" name="adminVarNameWidget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="spacing">
       <number>4</number>
      </property>
      <property name="margin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="varNameLabel">
        <property name="minimumSize">
         <size>
          <width>133</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>Имя переменной раздела</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="varNameEdit"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>482</width>
        <height>326</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="margin">
        <number>4</number>
       </property>
       <item>
        <widget class="QWidget" name="hideWidget" native="true">
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <property name="spacing">
           <number>4</number>
          </property>
          <property name="margin">
           <number>0</number>
          </property>
          <item>
           <widget class="QPushButton" name="hideButton">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>8</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="../../Resources.qrc">
              <normaloff>:/images/Up.png</normaloff>:/images/Up.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="variableWidget" native="true">
            <layout class="QVBoxLayout" name="variableLayout">
             <property name="spacing">
              <number>4</number>
             </property>
             <property name="margin">
              <number>0</number>
             </property>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="componentLayout">
         <property name="spacing">
          <number>4</number>
         </property>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>0</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../Resources.qrc"/>
 </resources>
 <connections/>
</ui>
